utils/appliance/db.py

```diff
@@ -137,7 +137,12 @@ def setup(self, **kwargs):
         if self.appliance.version != LATEST:
             # We only execute this on downstream appliances.
             # TODO: Handle external DB setup. Probably pop the db_address and decide on that one.
-            self.enable_internal(**kwargs)
+            self.enable_internal(
```

Comment:
Not that I know of. I did consider adding **kwargs to enable_internal, which is a simple safer way. Thoughts?